adele-booking/src/main/java/com/microservicesteam/adele/booking/infrastucture/config/GuavaModuleConfig.java

```diff
@@ -0,0 +1,16 @@
+package com.microservicesteam.adele.booking.infrastucture.config;
+
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.datatype.guava.GuavaModule;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class GuavaModuleConfig {
+
+    @Bean
+    public Module guavaModule() {
```

Comment:
@kelemena I haven't had a chance to check it, how we use this on the project, but this is the "prettiest" way to make Spring to use the Guava (de)serialisers.